src/lib.rs

```diff
@@ -679,15 +679,9 @@ where
     }
 
     #[allow(clippy::many_single_char_names)]
-    /// Premultiplies current coordinate system by specified matrix.
-    ///
-    /// The parameters are interpreted as matrix as follows:
-    ///   [a c e]
-    ///   [b d f]
-    ///   [0 0 1]
-    pub fn set_transform(&mut self, a: f32, b: f32, c: f32, d: f32, e: f32, f: f32) {
-        let transform = Transform2D([a, b, c, d, e, f]);
-        self.state_mut().transform.premultiply(&transform);
+    /// Premultiplies current coordinate system by specified transform.
+    pub fn set_transform(&mut self, transform: &Transform2D) {
```

Comment:
ACAICS this changes public API (for the better, arguably). Could you add an entry to the ChangeLog please?

Reply:
Done